testscripts/ssim_test.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
```

Comment:
would you mind adding a description of this script as a comment here?  It can also include examples and maybe a help page?